packages/cli-repl/src/mongosh-repl.spec.ts

```diff
@@ -352,6 +352,20 @@ describe('MongoshNodeRepl', () => {
         await tick();
         expect(output).to.include('somelongvariable');
       });
+      it('autocompletes partial repl commands', async() => {
+        input.write('.e');
+        await tabtab();
```

Comment:
Haven't noticed these helpers before, nice method names 😄

Reply:
They haven’t been around for too long, but @mcasimir’s brought this up as an idea during PR review :)